remoteit/desktop
# [DESK-384] windows code signing

### Description

Made buildable on windows machine, along with full app signing using the EV signing cert.

### Checklist




### Test plan

On Windows.

1. Install another alpha release
2. Wait for update notice
3. Update 

### Ticket



<https://remoteit.atlassian.net/browse/DESK-384>
